sky/packages/sky/lib/src/animation/performance.dart

```diff
@@ -51,30 +51,30 @@ abstract class WatchableAnimationPerformance {
 /// may also take direct control of the timeline by manipulating [progress], or
 /// [fling] the timeline causing a physics-based simulation to take over the
 /// progression.
-class AnimationPerformance implements WatchableAnimationPerformance {
-  AnimationPerformance({ this.duration, double progress }) {
+class Performance implements PerformanceView {
```

Comment:
I have a small worry that it'll be easy to confuse this with e.g. CPU performance. Would it be better to shorten it as "Animation" instead? I don't feel that strongly, though.